apps/condo/domains/ticket/schema/ExportTicketsService.js

```diff
@@ -64,8 +78,18 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
                 if (allTickets.length === 0) {
                     throw new GQLError(errors.NOTHING_TO_EXPORT, context)
                 }
+
+                const ticketsComments = await loadTicketCommentsForExcelExport({ ticketIds: allTickets.map(ticket => ticket.id) })
+
                 const excelRows = allTickets.map(ticket => {
-                    const comments = [...new Set(ticket.TicketComment || [])]
+                    const ticketComments = ticketsComments.filter(comment => comment.ticket === ticket.id)
+                    const organizationCommentsToRender = ticketComments
+                        .filter(comment => comment.type === ORGANIZATION_COMMENT_TYPE)
+                        .map(comment => renderComment(comment, locale))
+                    const residentCommentsToRender = ticketComments
+                        .filter(comment => comment.type === RESIDENT_COMMENT_TYPE)
+                        .map(comment => renderComment(comment, locale))
```

Comment:
Look like it's possible to decrease cycles count:
```suggestion
                    const organizationCommentsToRender = []
                    const residentCommentsToRender = []
                    ticketComments.forEach((ticketComment) => {
                        switch(comment.type) {
                            case ORGANIZATION_COMMENT_TYPE:
                                organizationCommentsToRender.push(renderComment(comment, locale))
                                break;
                            case ...
                        }
                    })
```